common/src/main/java/com/wynntils/functions/GuildFunctions.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.functions;
+
+import com.wynntils.core.components.Models;
+import com.wynntils.core.consumers.functions.Function;
+import com.wynntils.core.consumers.functions.arguments.FunctionArguments;
+import com.wynntils.models.players.type.GuildRank;
+import com.wynntils.utils.type.CappedValue;
+import java.util.List;
+
+public class GuildFunctions {
+    public static class CappedGuildLevelPercentageFunction extends Function<CappedValue> {
+        @Override
+        public CappedValue getValue(FunctionArguments arguments) {
+            return new CappedValue(Models.Guild.getGuildLevelPercentage(), 100);
+        }
+
+        @Override
+        protected List<String> getAliases() {
+            return List.of("capped_guild_level_percentage");
```

Comment:
This goes for all the functions, but the default for this will automatically be `capped_guild_level_percentage` so you don't need to do this.